### PR TITLE
Contacts component

### PR DESCRIPTION
**Pull request by GuilhermeJSilva**
* Includes the main page component for the contacts.
* The full component  is in src/components/wrapper.js.
* The component can be viewed [here](https://5d5d4830c5c9700008a7b43a--friendly-shockley-474156.netlify.com/contactos/)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,10 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.21",
```
Comment (pedromsfernandes) [inline package.json]: We already have React-Icons, which includes font awesome already

Comment (GuilhermeJSilva) [reply package.json]: Didn't notice, already ported to 'react-icons'.

File: src/components/contacts/contacts.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  icon: <FaMapMarkerAlt />,
+  href:
+    "https://www.google.com/maps/place/FEUP+-+Faculdade+de+Engenharia+da+Universidade+do+Porto/@41.17794,-8.597688,15z/data=!4m5!3m4!1s0x0:0x8b1e4a0bcdacc840!8m2!3d41.1779401!4d-8.5976876?hl=en-US",
+  text: "R. Dr. Roberto Frias, 4200-465 Porto, Portugal",
+}
+
+const Contacts = () => (
+  <div>
+    <Col className={ContactStyles.title_container}>
+      <h2 className={ContactStyles.title}>Contacta-nos:</h2>
```
Comment (smbea) [inline src/components/contacts/contacts.js]:
```suggestion
      <h4 className={ContactStyles.title}>Contacta-nos:</h4>
```

File: src/styles/contacts.module.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.title {
+  text-align: center;
+  font-weight: bold;
+}
+
+.title_bar {
+  width: 4em;
+  height: 0.3em;
```
Comment (smbea) [inline src/styles/contacts.module.css]:
```suggestion
  height: 0.2em;
```

File: src/styles/contacts.module.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  background-color: white;
+  border-top: 0;
+  margin: 0.5em auto 1em auto;
```
Comment (smbea) [inline src/styles/contacts.module.css]:
```suggestion
  margin: 0.5em auto 2em auto;
```

File: src/styles/contacts.module.css [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.link:hover,
+.location_link:hover {
+  text-decoration: none;
+  color: white;
+}
+
+.icon {
+  margin-right: 0.3em;
```
Comment (smbea) [inline src/styles/contacts.module.css]:
```suggestion
  margin-right: 0.6em;
```

File: src/styles/contacts.module.css [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  font-size: 1.8em;
```
Comment (smbea) [inline src/styles/contacts.module.css]:
```suggestion
  font-size: 1.6em;
```

File: src/components/contacts/contacts.js [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+        </Col>
+        <Col
+          className={ContactStyles.info_col + " " + ContactStyles.location_col}
+        >
+          <a href={location} className={ContactStyles.location_link}>
+            <span className={ContactStyles.icon}>
+              <FaMapMarkerAlt />
+            </span>
+            FEUP <br />
+            R. Dr. Roberto Frias, 4200-465 Porto, Portugal
```
Comment (smbea) [inline src/components/contacts/contacts.js]: Can you put each section that is now currently separated by the colon in its own line? I just think it makes everything look more centered

**Pull request merged**